code/drasil-example/Drasil/SWHS/Unitals.hs

```diff
@@ -446,6 +447,8 @@ tank_length_max = mkDataDef (unitary "tank_length_max"
   (nounPhraseSP "maximum length of tank")
   (sub (eqSymb tank_length) (Atomic "max")) metre Rational) 50
 
+frac_min_aux    = mkDataDef frac_min $ (dbl 1.0) * (10 $^ (negate 6))
```

Comment:
We should probably keep this in mind for #941 as we don't want scientific notation hacks.

Reply:
Ok. After scientific notation is done. I will check and correct all hacks.